_maps/superbowserstrackway/superbowserstrackway.yaml

```diff
@@ -0,0 +1,187 @@
+---
+name:
+  en: Super Bowser's Trackway
+desc:
+  en: In his many failed attempts to stomp out Mario, king Bowser has been known to transform into what can only be described as a giant super version of himself, plodding all around the Mushroom Kingdom leaving dinosour like footprints behind. This one's for sale!!  
+ruleSet: Standard
+theme: Mario
+initialCash: 2500
+targetAmount: 20000
+baseSalary: 300
+salaryIncrement: 275
+maxDiceRoll: 6
+frbFiles:
+ - superbowserstrackway
+background: forestFairies
+mapIcon: sbowsersICON
+music:
+  download:
+  - https://nikkums.io/cswt/BGM/superbowserstrackway.music.zip
+  - https://drive.google.com/file/d/1P-Tq8jVU0cuqEpHoA-6P69RvTU_b1uUR/view?usp=sharing

```

Comment:
This one should be `https://drive.google.com/uc?export=download&confirm=1&id=1P-Tq8jVU0cuqEpHoA-6P69RvTU_b1uUR`